internal/providers/terraform/azure/testdata/frontdoor_firewall_policy_test/frontdoor_firewall_policy_test.tf

```diff
@@ -0,0 +1,95 @@
+provider "azurerm" {
+  skip_provider_registration = true
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-rg"
+  location = "westus"
+}
+
+resource "azurerm_frontdoor_firewall_policy" "firewall_policy_example" {
+  name                = "Example"
+  resource_group_name = azurerm_resource_group.example.name
+
+  custom_rule {
+    name     = "Rule1"
+    enabled  = true
+    priority = 1
+    type     = "MatchRule"
+    action   = "Block"
+
+    match_condition {
+      match_variable = "RemoteAddr"
+      operator       = "IPMatch"
+      match_values   = ["192.168.1.0/24", "10.0.0.0/24"]
+    }
+  }
+
+  custom_rule {
+    name     = "Rule2"
+    enabled  = false
+    priority = 2
+    type     = "MatchRule"
+    action   = "Block"
+
+    match_condition {
+      match_variable = "RemoteAddr"
+      operator       = "IPMatch"
+      match_values   = ["192.168.1.0/24"]
+    }
+  }
+
+  managed_rule {
+    type    = "DefaultRuleSet"
+    version = "1.0"
+  }
+
+  managed_rule {
+    type    = "RuleSet2"
+    version = "1.0"
+  }
+}
+
+resource "azurerm_frontdoor_firewall_policy" "firewall_policy_with_usage" {
+  name                = "ExampleWithUsage"
+  resource_group_name = azurerm_resource_group.example.name
+
+  custom_rule {
+    name     = "Rule1"
+    enabled  = true
+    priority = 1
+    type     = "MatchRule"
+    action   = "Block"
+
+    match_condition {
+      match_variable = "RemoteAddr"
+      operator       = "IPMatch"
+      match_values   = ["192.168.1.0/24", "10.0.0.0/24"]
+    }
+  }
+
+  custom_rule {
+    name     = "Rule2"
+    enabled  = false
```

Comment:
I can't find the information if disabled custom rules are billed or not. As example we don't count the disabled tests for [azurerm_application_insights_web_test](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/application_insights_web_test), [infracost resource](https://github.com/infracost/infracost/blob/master/internal/providers/terraform/azure/application_insights_web_t.go#L23).

Reply:
This took a while to hunt down: https://docs.microsoft.com/en-us/azure/frontdoor/front-door-faq#will-i-be-billed-for-the-azure-front-door-resources-that-are-disabled-

> Azure Front Door resources, like Front Door profiles, routing rules are not billed in disabled. WAF policies and rules are billed even if disabled.

So the `azurerm_frontdoor_firewall_policy` rules should still have a cost if disabled, but the `azurerm_frontdoor` rules shouldn't.

Reply:
Thank you! Your documentation kung-fu is strong :D